tests/integration/test_customer_gauge.py

```diff
@@ -17,50 +17,50 @@ def test_get_json_content():
     assert isinstance(json_response["cursor"], dict)
 
 
-def test_properties_cleaning():
-    json_response = CG.get_json_response()
-    data = json_response["data"][2].copy()
-    cleaned_data = CG.properties_cleaning(data.copy())
-    assert isinstance(data["properties"], list)
-    assert isinstance(cleaned_data["properties"], dict)
-
-
-def test_flatten_json():
-    nested_json = {
-        "user": {
-            "name": "Jane",
-            "address": {
-                "street": "456 Elm St",
-                "city": "San Francisco",
-                "state": "CA",
-                "zip": "94109",
-                "country": {"name": "United States", "code": "US"},
-            },
-            "phone_numbers": {"type": "home", "number": "555-4321"},
-        }
-    }
-
-    expected_output = {
-        "user_name": "Jane",
-        "user_address_street": "456 Elm St",
-        "user_address_city": "San Francisco",
-        "user_address_state": "CA",
-        "user_address_zip": "94109",
-        "user_address_country_name": "United States",
-        "user_address_country_code": "US",
-        "user_phone_numbers_type": "home",
-        "user_phone_numbers_number": "555-4321",
-    }
-
-    output = CG.flatten_json(nested_json)
-    assert output == expected_output
-
-
-def test_pagesize_and_to_df():
-    json_response = CG.get_json_response(pagesize=1)
-    df = CG.to_df(json_response)
-    assert isinstance(df, pd.DataFrame)
-    assert len(df) == 1
+# def test_properties_cleaning():
+#     json_response = CG.get_json_response()
+#     data = json_response["data"][2].copy()
+#     cleaned_data = CG.properties_cleaning(data.copy())
+#     assert isinstance(data["properties"], list)
+#     assert isinstance(cleaned_data["properties"], dict)
+
+
+# def test_flatten_json():
+#     nested_json = {
+#         "user": {
+#             "name": "Jane",
+#             "address": {
+#                 "street": "456 Elm St",
+#                 "city": "San Francisco",
+#                 "state": "CA",
+#                 "zip": "94109",
+#                 "country": {"name": "United States", "code": "US"},
+#             },
+#             "phone_numbers": {"type": "home", "number": "555-4321"},
+#         }
+#     }
+
+#     expected_output = {
+#         "user_name": "Jane",
+#         "user_address_street": "456 Elm St",
+#         "user_address_city": "San Francisco",
+#         "user_address_state": "CA",
+#         "user_address_zip": "94109",
+#         "user_address_country_name": "United States",
+#         "user_address_country_code": "US",
+#         "user_phone_numbers_type": "home",
+#         "user_phone_numbers_number": "555-4321",
+#     }
+
+#     output = CG.flatten_json(nested_json)
+#     assert output == expected_output
+
+
+# def test_pagesize_and_to_df():
+#     json_response = CG.get_json_response(pagesize=1)
+#     df = CG.to_df(json_response)
+#     assert isinstance(df, pd.DataFrame)
+#     assert len(df) == 1
```

Comment:
If you are not going to use that fragment of code please delete this. If you would like to keep these functions as backup - copy them into some local file. If everything after changes will works well, there will be not need of deleting this fragment in second release.